Formula/ripgrep.rb

```diff
@@ -15,11 +15,13 @@ class Ripgrep < Formula
   depends_on "asciidoc" => :build
   depends_on "docbook-xsl" => :build
   depends_on "rust" => :build
+  depends_on "pkg-config" => :build
+  depends_on "pcre2" => :build
```

Comment:
You need to remove `:build` since this is a runtime dependency.

Reply:
Done, thanks for the suggestion.